recipe_scrapers/rezeptwelt.py

```diff
@@ -9,19 +25,69 @@ def host(cls):
         return "rezeptwelt.de"
 
     def site_name(self):
-        raise StaticValueException(return_value="Rezeptwelt")
+        return "Thermomix Rezeptwelt"
```

Comment:
```suggestion
        raise StaticValueException(return_value="Thermomix Rezeptwelt")
```

I admit this is a slightly unusual pattern that we use; it is used so that the interface of the library can indicate whether values were retrieved from the source HTML or whether they are static/constant values returned by the code.